README.md

```diff
@@ -1,19 +1,62 @@
-# ball-bearings-with-quarkus
+# ball-bearings-with-quarkus Project
```

Comment:
There are a lot of conflicts with your existing README.md

Please make sure, that you keep your origin version and add content on how to start the project.

Describe the prerequisits like JDK, gradle, etc. ... It is good practice to give precise information on how to set up the environment for developing.

Reply:
I copied the contribution guide to a new file "contributing.md" and added a link in README.md

The README.md contains information about starting and building the application. 

README.md contains now the section "Prerequisites"